ci/terraform/oidc/account-interventions.tf

```diff
@@ -13,6 +13,7 @@ module "frontend_api_account_interventions_role" {
     aws_iam_policy.dynamo_client_registry_read_access_policy.arn,
     aws_iam_policy.redis_parameter_policy.arn,
     module.oidc_txma_audit.access_policy_arn,
+    aws_iam_policy.dynamo_auth_session_read_policy.arn,
```

Comment:
This (and all new policies) needs to be added at the end of the list of policies in auth's roles. If the order is changed, the role is rewritten while live (not ideal infrastructure), meaning we'd have brief downtime on this lambda while it's permissions were updated. Concat can take multiple arguments, so it's best to have this added after the `ticf_cri_lambda_invocation_policy` just below.

Reply:
done